### ISSUE TITLE
Multiple Series Usage

### ISSUE THREAD

**Issue by NoBet41** [ankane/chartkick#250]
I have 2 simple questions:

1) Is it possible de-select all data from the graph while working with multiple series? The graph is becoming a bit messy if the number of data to be used is large.

2) What is the correct syntax to arrange the colors of data used in multiple-series?

Thanks, in advance.

**Comment by memoht** [comment]
1. I just created a chart using two series and was able to turn off each item in the series so answer to 1 is I think yes.
2. I loaded colors via an initializer and it seems follow using the color in the order they appear, and if it needs to repeats from first to last. This at least makes it easier to predict what color an item will be if you know what you are charting.
He provides a great example of a series chart here at https://github.com/ankane/chartkick#data. Also note, when I used Google charts I couldn't turn off items, but switching to Chartick and Highcharts together is a win.
